x/contractmanager/keeper/grpc_query_failure_test.go

```diff
@@ -127,6 +128,10 @@ func TestFailureQueryPaginated(t *testing.T) {
 			nullify.Fill(resp.Failures),
 		)
 	})
+	t.Run("More than limit", func(t *testing.T) {
```

Comment:
please stick with the test cases naming pattern: More than limit -> MoreThanLimit

Reply:
fixed